src/clj_kondo/impl/hooks.clj

```diff
@@ -19,9 +19,9 @@
 
 (defn reg-keyword!
   ([k]
-   (reg-keyword! k true))
+   (reg-keyword! k nil))
```

Comment:
it didn't seems right to register the keyword as a `:reg true` if user just `reg-keyword! :kw` without the intention of register that keyword as a definition, for those cases the 2 arity seems the correct approach.

Reply:
Yes, so this arity can be removed? Would it be useful for LSP to also have `ref-keyword` so you can find if `re-frame.core/subscribe` or `re-frame.core/dispatch` (specifically one of those) uses the keyword? Or is this covered by `find-references` well enough?

Reply:
Yeah, it's not necessary, the find references seems enough

Reply:
ok, let's remove it then

Reply:
@borkdude not sure we should remove the 1 arity, imagine user wants to reg a keyword but not necessarily it's a keyword that is a definition, so just `(reg-keyword :my-key)` looks valid, otherwise user would always register the keyword with the `:def`

Reply:
Why would a user ever want to "register" a keyword without any context? And didn't you rename that key to `:reg`?

Reply:
ops, yeah, `:reg` 😅.
You are right, if user don't register it, clj-kondo will reg as a normal keyword anyway.
I'll remove it so